downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/MapOldStableIds.java

```diff
@@ -0,0 +1,123 @@
+package org.reactome.release.downloadDirectory;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.model.ReactomeJavaConstants;
+
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.StandardOpenOption;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.HashSet;
+
+public class MapOldStableIds {
+	private static final Logger logger = LogManager.getLogger();
+	
+	private static Connection connect = null;
+	private static Statement statement = null;
+	private PreparedStatement preparedStatement = null;
+	private static ResultSet resultSet = null;
```

Comment:
These could be declared in the execute method

Reply:
*should

Reply:
...the logger is fine where it is. The connection object actually is also OK as a class-level static variable.

Reply:
I'm not sure I understand the reasoning for this? I'll make the change but I'm curious

Reply:
The reason is just to keep the scope of the variables as small as possible.  

Putting them in the method communicates they are only used there and also shows precisely where they are intialized.  There's no need to search if they are used or modified elsewhere and prevents them from accidentally being accessed by code in other methods (present or future) that shouldn't see them.  The idea is just that the smaller the scope, the smaller the section of code you have to keep track of them.